.eslintrc.js

```diff
@@ -69,6 +69,14 @@ module.exports = {
           'error', {
             definedTypes: ['NodeListOf']
           }
+        ],
+
+        // Maintain new line after description
+        'jsdoc/tag-lines': [
+          'warn',
+          'never', {
+            startLines: 1
```

Comment:
For some reason [`eslint-plugin-jsdoc`](https://www.npmjs.com/package/eslint-plugin-jsdoc) is now warning us about the new lines it (previously) wanted between JSDoc descriptions and tags

We were happy with them before so let's keep them for now